src/app.md

```diff
@@ -725,6 +726,55 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-semver-latest]: #ex-semver-latest
+<a name="ex-semver-latest"></a>
+## Find the latest version satisfying given range
+[![semver-badge]][semver] [![cat-config-badge]][cat-config]
+
+Given a list of version `&str`s, find the latest [`semver::Version`] that satisfies the given range
+"<= 1.0.0". A [`semver::VersionReq`] is used to compare the parsed version to a 
```

Comment:
I'd probably try to keep the description more compact. Haw about something along these lines?:

Given a list of version `&str`s, finds the latest [`semver::Version`] that satisfying given [`semver::VersionReq`] "<= 1.0.0" using [`VersionReq::matches`].